### PR TITLE
NC-606

### PR DESCRIPTION
**Pull request by root-beer**
ref: https://skyarch.backlog.jp/view/NC-606

### CODE REVIEW

**Comment by root-beer** [comment]
Please review and merge PR. @Joeper214 :bow:

**Comment by Joeper214** [comment]
I'll review this on monday. @root-beer . :bow:

**Comment by Joeper214** [comment]
Reviewing. ..

File: app/jobs/serverspec_job.rb [hunk 1]
```diff
@@ -25,16 +25,17 @@ def perform(physical_id, infra_id, user_id, serverspec_ids: nil, auto_generated:
       serverspec_ids = resource.all_serverspec_ids
     end
 
-    ws = WSConnector.new('notifications', User.find(user_id).ws_key)
+    @ws = WSConnector.new('notifications', User.find(user_id).ws_key)
 
+    infra_logger_serverspec_start(physical_id, infra_id, user_id, auto_generated, serverspec_ids)
```
Comment (Joeper214) [inline app/jobs/serverspec_job.rb]: :+1:

**Comment by Joeper214** [comment]
LGTM! Thanks. :bow:

**Pull request merged**